### ISSUE TITLE
loopserver image broken for versions > v0.9.52-beta

### ISSUE THREAD

**Issue by endline** [lightninglabs/loop#523]
When running the `loopserver` docker image found [here](https://hub.docker.com/r/lightninglabs/loopserver/tags) all versions greater than `0.9.52-beta` exit immediately upon startup.

**Expected behavior**

Expected to see the logs as per previous versions. At a minimum the below:

```
$ docker run --rm lightninglabs/loopserver:v0.9.52-beta daemon
2022-08-31 17:38:19.851 [INF] SRVR: Starting loop server version: 0.9.52-beta, commit: regtest, commit hash: regtest, go version: go1.17.3
2022-08-31 17:38:19.852 [INF] LNDC: Creating lnd connection to localhost:10009
...
```

**Actual behavior**

Image quits immediately, with no logs produced.

```
$ docker run --rm lightninglabs/loopserver:v0.9.54-beta daemon

~ $ echo $?
1
```

**To reproduce**

Run any version > 0.9.52 and you will see the image exit immediately.

**System information**

Fails to run as a standalone container (expected given no other services are running, but appears to panic instantly). Also fails when other services are running and passed correctly with flags i.e.

```
$ docker pull lightninglabs/loopserver:latest
$ docker run -d \
    -p 11009:11009 \
    -v /some/dir/to/lnd:/root/.lnd \
    lightninglabs/loopserver:latest \
      daemon \
      --maxamt=5000000 \
      --lnd.host=some-lnd-node:10009 \
      --lnd.macaroondir=/root/.lnd/data/chain/bitcoin/regtest \
      --lnd.tlspath=/root/.lnd/tls.cert
```

**Comment by levmi** [comment]
This should now be fixed. Are you able to get this working or still having issues?

**Comment by thinkmassive** [comment]
Apologies, the fix was merged last week but we never triggered the image build.

The fix is now released in [`lightninglabs/loopserver:v0.9.61-beta`](https://hub.docker.com/layers/lightninglabs/loopserver/v0.9.61-beta)
